packages/components/src/FilterChips/FilterChips.js

```diff
@@ -78,13 +79,15 @@ FilterChips.propTypes = {
     ),
     onDelete: PropTypes.func,
     onDeleteGroup: PropTypes.func,
-    deleteTitle: PropTypes.node
+    deleteTitle: PropTypes.node,
+    showDeleteButton: PropTypes.oneOf([ 'default', true, false ])
```

Comment:
It's kinda strange mixing boolean values and string. Can you use `null` or `undefined` instead of this? I think that the `undefined` will be just awesome! Because we won't have to introduce new default prop and the line 53 would change to:
```JS
{ (showDeleteButton === true || (showDeleteButton === undefined && filters.length > 0)) &&
```